### PR TITLE
add #range* predicates + update k and proofs submodules

### PR DESCRIPTION
**Pull request by daejunpark**

### CODE REVIEW

File: edsl.md [hunk 1]
```diff
@@ -270,5 +270,19 @@ Specifically, `#hashedLocation` is defined as follows, capturing the storage lay
...
+```
+
+### Utilities
+
+```k
+    syntax Bool ::= "#range" "(" Int "<"  Int "<"  Int ")" [function]
+                  | "#range" "(" Int "<"  Int "<=" Int ")" [function]
+                  | "#range" "(" Int "<=" Int "<"  Int ")" [function]
+                  | "#range" "(" Int "<=" Int "<=" Int ")" [function]
+ // -----------------------------------------------------
```
Comment (ehildenb) [inline edsl.md]: Formatting, extend the `----` to the end of `[function]`.

**Comment by daejunpark** [comment]
@ehildenb Ah you're right. I was worried about the performance, but shouldn't be that problem. Will revise it.

**Comment by ehildenb** [comment]
Jenkins: test mantis

**Comment by daejunpark** [comment]
@ehildenb please re-review and merge if it looks good to you

**Pull request merged**